ftag/flavours.yaml

```diff
@@ -58,12 +58,12 @@
   cuts: ["R10TruthLabel_R22v1 == 12"]
   colour: "#B45F06"
   category: xbb
-- name: top
-  label: Top
+- name: tqqb
+  label: Fully Contained Top
```

Comment:
I also probably would not have changed the label name here. What do @zcapjdb @philippgadow think?

Reply:
This can be changed after I don't mind, just as long as we have a separate label for each type. I realise now that, it's probably better to put the tqqb label in a separate extended double b-tagging category due to how some things are read elsewhere. I can update this in the #55 PR as this was an issue there